sound/soc/intel/boards/sof_sdw.c

```diff
@@ -1414,6 +1415,7 @@ static int create_sdw_dailink(struct snd_soc_card *card, int *link_index,
 			++codec_dlc_index;
 			++*codec_conf_index;
 		}
+		j = 0;
```

Comment:
Right, this answers, can you put a comment that we are resetting the counter for consecutive links to be scanned from the beginning?